src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -116,22 +120,38 @@
     public static final Achievement achievementUpgradeFisherMax = new MineColoniesAchievement("upgrade.fisher.max", "upgrade.fisher.max", 2, 5, ModBlocks.blockHutFisherman,
                                                                                                achievementBuildingFisher).registerStat();
 
+    /**
+     * Upgrade a farmer to lv 1.
+     */
+    public static final Achievement achievementBuildingFarmer   = new MineColoniesAchievement("upgrade.farmer.first", "upgrade.fisher.first", 0, 6, ModBlocks.blockHutFarmer,
+                                                                                               achievementGetSupply).registerStat();
+    /**
+     * Max out a farmer.
+     */
+    public static final Achievement achievementUpgradeFarmerMax = new MineColoniesAchievement("upgrade.farmer.max", "upgrade.fisher.max", 2, 6, ModBlocks.blockHutFarmer,
+                                                                                               achievementBuildingFarmer).registerStat();
+
     // Sizes
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_SETTLEMENT} citizens.
      */
-    public static final Achievement achievementSizeSettlement = new MineColoniesAchievement("size.pioneer", "size.settlement", 2, -2, Items.iron_ingot, null).registerStat();
+    public static final Achievement achievementSizeSettlement = new MineColoniesAchievement("size.pioneer", "size.settlement", 2, -2, ModItems.itemAchievementProxySettlement, null).registerStat();
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_TOWN} citizens.
      */
-    public static final Achievement achievementSizeTown       = new MineColoniesAchievement("size.town", "size.town", 4, -2, Items.gold_ingot, achievementSizeSettlement)
+    public static final Achievement achievementSizeTown       = new MineColoniesAchievement("size.town", "size.town", 4, -2, ModItems.itemAchievementProxyTown, achievementSizeSettlement)
                                                                   .registerStat();
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_CITY} citizens.
      */
-    public static final Achievement achievementSizeCity       = new MineColoniesAchievement("size.city", "size.city", 6, -2, Items.diamond, achievementSizeTown).registerStat();
+    public static final Achievement achievementSizeCity       = new MineColoniesAchievement("size.city", "size.city", 6, -2, ModItems.itemAchievementProxyCity, achievementSizeTown).registerStat();
+
+    /**
+     * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_METROPOLIS} citizens.
+     */
+    public static final Achievement achievementSizeMetropolis = new MineColoniesAchievement("size.metropolis", "size.metropolis", 8, -2, ModItems.itemAchievementProxyMetropolis, achievementSizeCity).registerStat();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 214 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)